core/templates/components/summary-tile/exploration-summary-tile.component.html

```diff
@@ -82,7 +90,8 @@ <h3 class="activity-title protractor-test-exp-summary-tile-title">
       </div>
       <div class="mobile-activity-card-details">
         <h3 class="mobile-exploration-title protractor-test-exp-summary-tile-title">
-          <span>{{ explorationTitle | truncate:40 }}</span>
+          <span *ngIf="!isHackyExplTitleTranslationDisplayed()">{{ explorationTitle | truncate:40 }}</span>
+          <span *ngIf="isHackyExplTitleTranslationDisplayed()">{{ explTitleTranslationKey | translate | truncate:40 }}</span>
```

Comment:
Here and elsewhere: perhaps use exploration instead of expl? Or, alternatively, exp, since that's what we use in the backend (e.g. exp_services).

Reply:
Made it exp, I just wanted to shorten this up as exploration makes the whole thing very big in length.